vhdbuilder/packer/test/run-test.sh

```diff
@@ -110,8 +111,7 @@ else
       --public-ip-address "" \
       ${TARGET_COMMAND_STRING}
 fi
-
-echo "VHD test VM username: $TEST_VM_ADMIN_USERNAME, password: $TEST_VM_ADMIN_PASSWORD"
```

Comment:
I can't remember, isn't there a debug mode where we let the user keep the VM for debugging? if so then we want to upload an ssh key or something to build artifacts

though, I think at the end of the day devs can just reset the password on the VM manually